code/modules/power/groundmap_geothermal.dm

```diff
@@ -248,6 +236,7 @@ GLOBAL_VAR_INIT(generators_on_ground, 0)
 		playsound(loc, 'sound/items/weldingtool_weld.ogg', 25)
 		user.visible_message(span_notice("[user] carefully starts burning [src]'s resin off."),
 		span_notice("You carefully start burning [src]'s resin off."))
+		user.balloon_alert(user, "You carefully start burning [src]'s resin off.")
```

Comment:
ditto, also sidenote if you're adding balloon alerts remove the visisble messages